lib/katello/engine.rb

```diff
@@ -87,6 +87,13 @@ def find_assets(args = {})
       require 'katello/apipie/validators'
     end
 
+    # make sure the Katello plugin is initialized before `after_initialize`
+    # hook so that the resumed Dynflow tasks can rely on everything ready.
+    initializer 'katello.register_plugin', :before => :finisher_hook do
```

Comment:
One more thought, we need this to run before `foreman_tasks.initialize_dynflow` initializer is run? If so, this might could be done more explicitly via:

```
initializer 'katello.register_plugin', :before => 'foreman_tasks.initialize_dynflow'
```